.idea/sonarlint/issuestore/2/d/2d3f511e863340cc1cb4f84b8f88e1a7b1f53722

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Check your .gitignore file. The files under .idea should not reside in repository